bin/website/tests.js

```diff
@@ -352,3 +372,20 @@ ava.serial('website snapshot receive to zfs', async t => {
 
     await tests.runProcess('zfs destroy -r h1_cli_test');
 });
+
+ava.serial('website restart nodejs app', async t => {
+    const password = await tests.getToken();
+    const image = 'h1cr.io/website/node:12';
+    const website = await tests.run(`website create --name ${tests.getName(t.title)} --type website-dedicated --image '${image}' --password ${password}`);
+    const content = images[image].code;
+    await mkDirWebsite(website, { password}, 'app');
+    await putFileWebsite(website, { password }, 'app/index.js', content);
+    await tests.run(`website restart --website ${website.id}`);
+    await tests.delay(5 * 1000);
```

Comment:
can we extract and name those time constants?

Reply:
Yes, of course, we can.